traits/has_traits.py

```diff
@@ -305,6 +306,16 @@ def _property_method(class_dict, name):
     return class_dict.get(name)
 
 
+def _mapped_trait_default_method(trait, trait_name):
+
+    def default_method(key_trait, key_trait_name, object):
+        return key_trait.handler.mapped_value(
+            getattr(object, key_trait_name, Undefined)
+        )
+
+    return partial(default_method, trait, trait_name)
```

Comment:
I'm assuming that the `partial` use was necessary to make the default method is pickleable. Is that right?

Reply:
No, that was just a remnant from when I was still playing with `lambda`. Pickling was working without it as well.